### PR TITLE
UMA - 41 "Configuration of Access Rights in Role Management"

### PR DESCRIPTION
**Pull request by manojkrishna1991**

### CODE REVIEW

File: src/multi-level-checkbox-editor/node-box.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ interface Props {
     onNodeClick: (node: Node) => void;
     setNodeValue: (node: Node, value: boolean) => void;
     levelHeaderLabel: string;
+    checkBoxHeader?: boolean;
```
Comment (BenjaminEckardt) [inline src/multi-level-checkbox-editor/node-box.tsx]: What about a rename of `checkBoxHeader` to `disableHeaderCheckbox`?

Comment (MostafaBalata) [inline src/multi-level-checkbox-editor/node-box.tsx]: What about renaming this to `isCheckBoxHeaderDisabled`,`isCheckBoxHeaderVisable` or`isCheckBoxHeader` or any way that makes it questionable?

Comment (MostafaBalata) [reply src/multi-level-checkbox-editor/node-box.tsx]: [Deprecated comment] I didn't notice Benjamin has already commented the same,
https://github.com/Webtrekk/Ampel-UI/pull/58#discussion_r389734890

Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/node-box.tsx]: yeah looks nice will change it

Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/node-box.tsx]: no problem

**Pull request merged**